homeassistant/components/reolink/number.py

```diff
@@ -62,10 +63,111 @@ class ReolinkNumberEntityDescription(
+        icon="mdi:spotlight-beam",
+        entity_category=EntityCategory.CONFIG,
+        native_step=1,
+        get_min_value=lambda api, ch: 1,
+        get_max_value=lambda api, ch: 100,
+        supported=lambda api, ch: api.supported(ch, "floodLight"),
+        value=lambda api, ch: api.whiteled_brightness(ch),
+        method=lambda api, ch, value: api.set_whiteled(ch, brightness=int(value)),
+    ),
+    ReolinkNumberEntityDescription(
+        key="volume",
+        name="Volume",
+        icon="mdi:volume-high",
+        entity_category=EntityCategory.CONFIG,
+        native_step=1,
+        get_min_value=lambda api, ch: 0,
+        get_max_value=lambda api, ch: 100,
+        supported=lambda api, ch: api.supported(ch, "volume"),
+        value=lambda api, ch: api.volume(ch),
+        method=lambda api, ch, value: api.set_volume(ch, volume=int(value)),
+    ),
+    ReolinkNumberEntityDescription(
+        key="guard_return_time",
+        name="Guard return time",
+        icon="mdi:crosshairs-gps",
+        entity_category=EntityCategory.CONFIG,
+        native_step=1,
+        native_unit_of_measurement=UnitOfTime.SECONDS,
+        get_min_value=lambda api, ch: 10,
+        get_max_value=lambda api, ch: 300,
+        supported=lambda api, ch: api.supported(ch, "ptz_guard"),
+        value=lambda api, ch: api.ptz_guard_time(ch),
+        method=lambda api, ch, value: api.set_ptz_guard(ch, time=int(value)),
+    ),
+    ReolinkNumberEntityDescription(
+        key="motion_sensitivity",
+        name="Motion sensitivity",
+        icon="mdi:motion-sensor",
+        entity_category=EntityCategory.CONFIG,
+        native_step=1,
+        get_min_value=lambda api, ch: 1,
+        get_max_value=lambda api, ch: 50,
+        supported=lambda api, ch: api.supported(ch, "md_sensitivity"),
+        value=lambda api, ch: api.md_sensitivity(ch),
+        method=lambda api, ch, value: api.set_md_sensitivity(ch, int(value)),
+    ),
+    ReolinkNumberEntityDescription(
+        key="ai_face_sensititvity",
+        name="AI face sensitivity",
+        icon="mdi:face-recognition",
+        entity_category=EntityCategory.CONFIG,
+        native_step=1,
+        get_min_value=lambda api, ch: 0,
+        get_max_value=lambda api, ch: 100,
+        supported=lambda api, ch: api.supported(ch, "ai_sensitivity")
+        and api.ai_supported(ch, "face"),
+        value=lambda api, ch: api.ai_sensitivity(ch, "face"),
+        method=lambda api, ch, value: api.set_ai_sensitivity(ch, int(value), "face"),
+    ),
+    ReolinkNumberEntityDescription(
+        key="ai_person_sensititvity",
+        name="AI person sensitivity",
+        icon="mdi:account",
+        entity_category=EntityCategory.CONFIG,
+        native_step=1,
+        get_min_value=lambda api, ch: 0,
+        get_max_value=lambda api, ch: 100,
+        supported=lambda api, ch: api.supported(ch, "ai_sensitivity")
+        and api.ai_supported(ch, "people"),
```

Comment:
Maybe this fits better

```suggestion
        supported=lambda api, ch: (
            api.supported(ch, "ai_sensitivity") and api.ai_supported(ch, "people")
        ),   
```